packages/api/README.md

```diff
@@ -36,7 +85,7 @@ The reindex command creates a new index and reindexes the source index to the ne
 
 An alias should not be specified unless you have a specific alias configured. If a source index is not specified, it will default to the index from the alias. If you want to name the destination index something particular, you can specify a name, otherwise the destination index name will default to 'cumulus-year-month-day' with today's date.
 
-```
+```bash
 cumulus-api reindex --help
```

Comment:
So you intend everything under here to be bash?

Reply:
Won't it go until the next one on line 108?

Reply:
All the places I added `bash` is just a Markdown linting thing. It's not valid Markdown to have a code block (```) without specifying the language in the block, otherwise the syntax highlighting and other things might be off.

I'm not sure I understand the comment about line 108. That is a separate code block but there is a header and some body text on lines 104 - 106.